src/main/java/org/fungover/storm/client/ClientHandler.java

```diff
@@ -14,9 +14,11 @@
     private OutputStream out;
     private BufferedReader in;
     private HttpResponseStatusCodes statusCode;
+    private FileRequestHandler fileRequestHandler;
 
-    public ClientHandler(Socket socket) {
+    public ClientHandler(Socket socket, FileRequestHandler fileRequestHandler) {
```

Comment:
Nice. It looks like you made fileRequestHandler a dependency injection so that you could test ClientHandler more easily.